tests/pom.xml

```diff
@@ -99,7 +99,7 @@ mvn test \
     <dependency>
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcpkix-jdk15on</artifactId>
-      <version>1.47</version>
+      <version>1.48</version>
```

Comment:
Given the idea is to update it probably makes sense to go for the latest version rather than this 3+ year old version (same goes for the other instance).